pkg/tar/tar.go

```diff
@@ -215,6 +222,22 @@ func (t *stiTar) ExtractTarStream(dir string, reader io.Reader) error {
 	}
 }
 
+func extractLink(dir string, header *tar.Header, tarReader io.Reader) error {
+	dest := filepath.Join(dir, header.Name)
+	source := header.Linkname
+
+	glog.V(3).Infof("Creating symbolic link from %s to %s", dest, source)
```

Comment:
Didn't know about `%q`, thanks! Is it `glog` specific modifier or it's something standard?